ironic/drivers/modules/drac/raid.py

```diff
@@ -727,6 +765,45 @@ def _filter_logical_disks(logical_disks, include_root_volume,
     return filtered_disks
 
 
+def _change_physical_disk_conversion(
+        node, mode=None, controllers_to_physical_disk_ids=None):
+    """
+    :param node: an ironic node objec
```

Comment:
Can you change "objec" to "object" in the line above?

Reply:
Done.